osmnx/speed.py

```diff
@@ -45,6 +46,8 @@ def add_edge_speeds(G, hwy_speeds=None, fallback=None, precision=1):
         values on any edge
     precision : int
         decimal precision to round speed_kph
+    agg : function
+        aggregation function to impute missing values from observed values
```

Comment:
Consider explicitly mentioning `np.median` as example of different input. Maybe "aggregation function to impute missing values from observed values. Use np.median if you are trying to impute official values on highways" or something along those lines.

Rationale: np.mean and np.median are only agg function that comes to my mind to be useful here. If so, other one should maybe have honorary mention:)

Reply:
Resolved in f645225